shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -69,6 +101,14 @@ protected int getNumOfArgs() {
 
   @Override
   public int run(CommandLine cl) throws IOException {
+    String[] args = cl.getArgs();
+    if (cl.hasOption("h")
```

Comment:
if the user specifies -h anywhere, we should probably print usage no matter what

Reply:
If -h appears after report capacity, I hope to print detailed capacityCommand usage.

Added comment:
```java
if (cl.hasOption("h")
        && !(args.length > 0 && args[0].equals("capacity"))) {
      // if category is capacity, we print report capacity usage inside CapacityCommand.
```

Reply:
should this be `||` instead of `&&`?

Reply:
we need a help option and the category is not capacity.

Reply:
I see, sgtm